### DISCUSSION TITLE
Error when en-core-web-sm is downloaded

### DISCUSSION THREAD

**Discussion by akesh1235** [explosion/spaCy#12613] [Help: Installation] [answered]
No compatible package found for 'en-core-web-sm' (spaCy v3.5.2)

**Comment by bdura** [comment, marked as answer]
Hello @akesh1235, could you retry the installation within a new environment? From your other threads, it looks like you want to use ERRANT, which is incompatible with spaCy v3. Could you try installing it from scratch, since it has the right spaCy requirements?

**Reply by akesh1235** [reply]
Yeah it worked
Thank you sir
